ConferencesApp/ConferenceTableViewController.swift

```diff
@@ -117,11 +130,107 @@ class ConferenceTableViewController: UITableViewController, MKMapViewDelegate {
 
         let theSpan:MKCoordinateSpan = MKCoordinateSpanMake(latDelta, longDelta)
         let venueLocation:CLLocationCoordinate2D = CLLocationCoordinate2DMake(latitude, longitude)
-                let theRegion:MKCoordinateRegion = MKCoordinateRegionMake(venueLocation, theSpan)
-
+        let theRegion:MKCoordinateRegion = MKCoordinateRegionMake(venueLocation, theSpan)
```

Comment:
Colon Violation: When specifying a type, always associate the colon with the identifier